doc/WebSite/Multi-Lingual-Entities.md

```diff
@@ -0,0 +1,89 @@
+### Introduction
+
+ASP.NET Boilerplate provides two basic interfaces for Multi-Lingual entity definitions. 
+
+#### IMultiLingualEntity
+
+IMultiLingualEntity interface represents is used to mark multi lingual entities. The entities marked with IMultiLingualEntity interface must define language-neutral information. The entities marked with IMultiLingualEntity contains a collection of Translations which contains language-dependent information.
```

Comment:
remove represents from "interface represents is used".
Also, writing IMultiLingualEntity as ``IMultiLingualEntity<TTranslation>`` (**bold** or ``code``) makes it more clear.